Core/Graphics/Source/DeviceImpl.cpp

```diff
@@ -82,6 +86,9 @@ namespace Babylon::Graphics
             case 16:
                 init.resolution.reset |= BGFX_RESET_MSAA_X16;
                 break;
+            default:
+                m_bgfxCallback.trace(__FILE__, __LINE__,  "WARNING: Setting an incorrect value for SetMSAA (%d). Correct values are 0, 1 (disable MSAA) or 2, 4, 8, 16.", int(value));
```

Comment:
It's a bit weird to rely on bgfx for API contract warnings, but okay for now. I wonder if throwing an exception is better?

Reply:
No strong opinion here. Throwing an exception here seems a bit too much. It may not be the rendering you want but it will work. To me an exception is needed when the behavior of the app will be (heavily) affected.